test/unit/classes/request-information-container.ts

```diff
@@ -67,27 +68,27 @@ describe('RequestInformationContainer', () => {
     const VALID_NUMBER_INPUT = 500;
     it('Should assign the value to the url property', () => {
       ric.setUrl(VALID_STRING_INPUT);
-      assert.deepEqual(ric.url, VALID_STRING_INPUT);
+      deepStrictEqual(ric.url, VALID_STRING_INPUT);
     });
     it('Should assign the value to the method property', () => {
       ric.setMethod(VALID_STRING_INPUT);
-      assert.deepEqual(ric.method, VALID_STRING_INPUT);
+      deepStrictEqual(ric.method, VALID_STRING_INPUT);
```

Comment:
These look like strings, no?  Why wouldn't they just be `assert.strictEquals`?

Reply:
I've updated it to use `assert.strictEquals`.